src/humancellatlas/data/metadata/api.py

```diff
@@ -306,11 +317,32 @@ def __init__(self,
         self.publications = set(ProjectPublication.from_json(publication)
                                 for publication in content.get('publications', []))
         self.contributors = {ProjectContact.from_json(contributor) for contributor in content.get('contributors', [])}
-        self.insdc_project_accessions = set(content.get('insdc_project_accessions', []))
-        self.geo_series_accessions = set(content.get('geo_series_accessions', []))
-        self.array_express_accessions = set(content.get('array_express_accessions', []))
-        self.insdc_study_accessions = set(content.get('insdc_study_accessions', []))
         self.supplementary_links = set(content.get('supplementary_links', []))
+        self.accessions = []
+        for key in content.keys():
+            if key.endswith('_accessions'):
```

Comment:
Don't duplicate the suffix. Either extract a variable or use rpartition('_') and test for 3rd return value to be the suffix.